### PR TITLE
make some variables in jlls const

### PR DESCRIPTION
**Pull request by KristofferC**

### CODE REVIEW

File: stdlib/CompilerSupportLibraries_jll/src/CompilerSupportLibraries_jll.jl [hunk 1]
```diff
@@ -57,7 +57,7 @@ function __init__()
     global libgomp_handle = dlopen(libgomp)
     global libgomp_path = dlpath(libgomp_handle)
     global artifact_dir = dirname(Sys.BINDIR)
-    global LIBPATH[] = dirname(libgcc_s_path)
+    LIBPATH[] = dirname(libgcc_s_path)
```
Comment (fingolfin) [inline stdlib/CompilerSupportLibraries_jll/src/CompilerSupportLibraries_jll.jl:60]: Just out of curiosity, why is this necessary (it is not mentioned in the PR description nor the commit message, as far as I could tell)

Comment (KristofferC) [reply stdlib/CompilerSupportLibraries_jll/src/CompilerSupportLibraries_jll.jl:60]: It is just a redundant `global` annotation. It doesn't do anything.

**Review by staticfloat — APPROVED** [review]
looks fine to me.

**Review by Sacha0 — APPROVED** [review]
